app-emulation/fs-uae-launcher/additional-files/fs-uae-launcher.rdef.in

```diff
@@ -0,0 +1,54 @@
+
+resource app_flags B_SINGLE_LAUNCH | B_ARGV_ONLY;
+
+resource app_version {
+	major  = @MAJOR@,
+	middle = @MIDDLE@,
+	minor  = @MINOR@,
+	variety = B_APPV_FINAL,
+	internal = 0,
+	short_info = "FS-UAE Launcher",
```

Comment:
You can add @SHORT@ var and get it from recipe SUMMARY.

Reply:
Ah, good idea - fixed.